x/ccv/provider/keeper/misbehaviour.go

```diff
@@ -134,10 +131,6 @@ func headerToLightBlock(h ibctmtypes.Header) (*tmtypes.LightBlock, error) {
 // CheckMisbehaviour checks that headers in the given misbehaviour forms
 // a valid light client attack and that the corresponding light client isn't expired
 func (k Keeper) CheckMisbehaviour(ctx sdk.Context, misbehaviour ibctmtypes.Misbehaviour) error {
-	if err := misbehaviour.ValidateBasic(); err != nil {
```

Comment:
Can you elaborate why this was removed?

Reply:
Checked https://github.com/cosmos/interchain-security/pull/826#discussion_r1291723790 for more context.